server/middleware/setUpMultipartFormDataParsing.ts

```diff
@@ -0,0 +1,25 @@
+import { Router, ErrorRequestHandler } from 'express'
+import multer, { MulterError } from 'multer'
+
+export default function setUpMultipartFormDataParsing(): Router {
+  const router = Router({ mergeParams: true })
+  const maxUploadSize = 10 * 1000 // 10kb
+  const upload = multer({ dest: 'uploads/', limits: { fileSize: maxUploadSize } })
+
+  router.use(upload.single('file'))
+  router.use(uploadedFileTooLargeHandler)
+
```

Comment:
Could possibly check the file mimetype too here? e.g. Check for .csv or .txt perhaps?  Not that it's much protection, but just avoids people trying to upload JPGs, PNGs etc..